examples/vite-react-avatar-stack-2/src/App.tsx

```diff
@@ -0,0 +1,79 @@
+import { useMemo, useEffect } from "react";
+import { SpaceProvider, SpacesProvider, useMembers, useSpace } from "@ably/spaces/react";
+import Spaces, { type SpaceMember } from "@ably/spaces";
+import { Avatar } from "./components/Avatar";
+import "./styles/avatars.css";
+
+export type Member = Omit<SpaceMember, "profileData"> & {
+  profileData: { memberColor: string; name: string };
+};
+
+const colors = ["#9951F5", "#f1c232", "#f44336"];
+const mockNames = [
+  "Anum Reeve",
+  "Tiernan Stubbs",
+  "Hakim Hernandez",
+];
+
+function AvatarStack() {
+  const name = useMemo(() => {
+    return mockNames[Math.floor(Math.random() * mockNames.length)];
+  }, []);
+  const memberColor = useMemo(() => {
+    return colors[Math.floor(Math.random() * colors.length)];
+  }, []);
```

Comment:
I would suggest moving this code with `colors` and `mockNames` into some utils file

Reply:
decided to skip this, to reduce number of files